app/code/Magento/Ui/Test/Mftf/Test/AdminDisappearErrorValidateMessageFieldTest.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<!-- Test XML Example -->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminDisappearErrorValidateMessageFieldTest">
+        <annotations>
+            <features value="Ui component"/>
+            <stories value="Error message"/>
+            <title value="Assert disappear error message."/>
+            <description value="Verify disappear error message when disabled field."/>
```

Comment:
If moving it to Bundle module, these tags should also reflect what actually the test does.